RestaurantsNearMe/Data/Interfaces/RestaurantListProving.swift

```diff
@@ -10,7 +10,8 @@ import Combine
 
 // sourcery: AutoMockable
 protocol RestaurantListProving {
```

Comment:
Nit: Should this be `RestaurantListProviding` (instead of `Proving`)?

Reply:
oh good catch 😅